src/App.java

```diff
@@ -98,6 +137,67 @@ public void actionPerformed(ActionEvent e) {
 
 		mainPanel.add(centerPanel, BorderLayout.CENTER);
 		// end center panel setup
+		
+		//South Panel
+		JPanel _buttonPanel = new JPanel();
+		_buttonPanel.setPreferredSize(new Dimension(200, 30));
+		_buttonPanel.setBackground(java.awt.Color.gray);
+		// Creating buttons
+			JButton _commitButton = new JButton("Commit");
+			JButton _pushButton = new JButton("Push");
+			JButton _pullButton = new JButton("Pull");
+		// Adds the buttons to the panel
+			_buttonPanel.add(_commitButton);
+			_buttonPanel.add(_pushButton);
+			_buttonPanel.add(_pullButton);
+		//Action Listener for the commit button
+			_commitButton.addActionListener(new ActionListener() {
+				@Override
+				public void actionPerformed(ActionEvent e) {
+					if (gitSubprocessClient == null) {
+					    showLoadFail();
+					}
+					else {
+						commitMessage = commitInputBox.getText();
+						gitSubprocessClient.gitCommit(commitMessage);
+						statusText.setText("Successfully committed: " + commitMessage);
```

Comment:
This line will be immediately covered by the call to updateGitStatus, but I think we should create another textbox to list out a log of changes that the user has made below the status box. Also, displaying line 162 instead of just the commit message would be better since it would also print any errors to the user.

Reply:
A similar issue also happens on line 178 and line 193 with the push and pull buttons.